File: EjerciciosC++/numeroParImpar/numeroParImpar/parImpar.cpp
#include <iostream>
using namespace std;

int main() {

	int numero;

	cout << "Introduzca un numero entero positivo: ";
	cin >> numero;

	if (numero <= 0) {

		cout << "\nEste numero no es positivo . . ." << endl;

	}
	else {

		if (numero % 2 == 0) {

			cout << "El numero " << numero << " es par" << endl;

		}
		else {
			
			cout << "El numero " << numero << " es impar" << endl;

		}

	}

	system("PAUSE");

}

File: Ejercicios C#/Par Impar/Par Impar/ParImpar.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Par_Impar
{
    class Par_Impar
    {
        static void Main()
        {

            int numero;

            Console.WriteLine("Ingrese un número entero positivo");
            numero = int.Parse(Console.ReadLine());

            if(numero <= 0)
            {
                Console.WriteLine("Este numero no es positivo");

            }else if(numero % 2 == 0)
            {

                Console.WriteLine("El numero {0} es par", numero);

            }
            else
            {

                Console.WriteLine("El numero {0} es impar", numero);

            }


            Console.Write("Presione una tecla para continuar ...");
            Console.Read();

        }
    }
}
